.github/workflows/build-and-deploy.yml

```
@@ -87,4 +87,4 @@ jobs:
         run: ssh webhost "echo ${{ secrets.PAT }} | docker login ghcr.io -u ${{ github.actor }} --password-stdin"
 
       - name: Docker Compose Up
-        run: ssh webhost "export AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }} && export AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }} && env && cd ~/apps/resume-builder && docker compose pull && docker compose up -d"
+        run: ssh webhost "export INFISICALCLIENTID=${{ secrets.INFISICALCLIENTID }} && export INFISICALCLIENTSECRET=${{ secrets.INFISICALCLIENTSECRET }} && env && cd ~/apps/resume-builder && docker compose pull && docker compose up -d"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)